### ISSUE TITLE
Disable JSCS auto-fix?

### ISSUE THREAD

**Issue by gzurbach** [godaddy/javascript#27]
We have the following npm script in our `package.json`:

``` json
"scripts": {
  "lint_local": "godaddy-js-style-eslint -c .eslintrc server common test && godaddy-js-style-jscs server common test",
  "lint_jenkins": "godaddy-js-style-eslint -c .eslintrc -f checkstyle server common test -o checkstyle-eslint.xml && godaddy-js-style-jscs -r checkstyle server common test > checkstyle-jscs.xml"
}
```

In Jenkins, we run `npm run lint_jenkins`. We expect that `checkstyle-eslint.xml` and `checkstyle-jscs.xml` will be reporting style issues. But instead because [`--fix` is set by default](https://github.com/godaddy/javascript/blob/master/bin/jscs#L11) by `godaddy-js-style-jscs`, our source files get fixed during the build. This is not the desired behavior. 

Would you guys accept a PR adding an option to disable auto-fix? I'm not sure how it would play with `fashion-show` (might be tricky). The change would be backward-compatible, and would allow us to stop our build if the style guide is not respected.

**Comment by indexzero** [comment]
Making this an option is definitely a feature that makes sense. The [relevant code is here](https://github.com/godaddy/javascript/blob/master/bin/jscs#L11). 

Something to consider w.r.t. your overall process: if you do this:

``` js
"scripts": {
  "pretest": "npm run lint_local",
  "test": "echo 'whatever your test script is'",
  "lint_local": "godaddy-js-style-eslint -c .eslintrc server common test && godaddy-js-style-jscs server common test",
  "lint_jenkins": "godaddy-js-style-eslint -c .eslintrc -f checkstyle server common test -o checkstyle-eslint.xml && godaddy-js-style-jscs -r checkstyle server common test > checkstyle-jscs.xml"
}
```

Then code will be auto-fixed before committing and getting to Jenkins.

**Comment by indexzero** [comment]
Fixed in #40.
